rspace/src/main/scala/coop/rchain/rspace/trace/TuplespaceEvent.scala

```diff
@@ -39,6 +38,7 @@ object TuplespaceEvent {
 
   def from(consume: Consume): Option[(Blake2b256Hash, TuplespaceEvent)] = consume match {
     case Consume(singleChannelHash :: Nil, _, _) =>
+      // ???????????????? it is possible that the consume is peek
```

Comment:
Do we have issue to track this?

Reply:
https://github.com/rchain/rchain/issues/3339
related